### DISCUSSION TITLE
Patreon not connected to Sauce, but payment has been made with card

### DISCUSSION THREAD

**Discussion by 040Noud** [SauceLLC/sauce4strava#187] [General]
Hi!

I have a small issue with the connection with Sauce. Payment has been made on the 1th of march as expected, but no subscription is visible in Patreon, and thus no connection in Sauce for Strava and Zwift. What to do? 

Hope you can help!

Noud.

**Comment by mayfield** [comment]
Hi @040Noud can you email me a screenshot of the error to support@sauce.llc?  Probably something simple like using one of the federated login buttons instead of email/password.

PS: Going to close this issue so we can discuss over email and not share any of your info.
